process/release_process.md

```diff
@@ -0,0 +1,155 @@
+# Release process
+
+Document version: **v1.0.0**
+
+Document status: **Released**
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Impacted domains](#impacted-domains)
+- [How to create a release?](#how-to-create-a-release)
+  - [xt-core](#xt-core)
+  - [moulin-based products](#moulin-based-products)
+- [Split of responsibilities](#split-of-responsibilities)
+- [Example of releasing the moulin-based project](#example-of-releasing-the-moulin-based-project)
+
+Please note that this table of contents is written in Markdown and will work best in a markdown-friendly environment. Hyperlinks in the table of content won't work here. They need to be associated with the actual sections in your markdown document. The text after the '#' should match exactly the title of the section.

```

Comment:
I would recommend to remove this sentence for the following reasons:
- Hyperlinks in the table of contents work fine, because they are relative to the current document
- File extension .md clearly states that this is markdown, and GitHub hosting for this document, handles it properly.

Reply:
Hi @rshym, it is a copy-paste from ChatGPT. I didn't even notice that I'd overtaken not only the table of contents but also the further part of its answer. Removed. Thanks for noticing this! ))